openapi-core/src/main/java/io/ballerina/openapi/core/generators/common/OASModifier.java

```diff
@@ -53,6 +52,7 @@
  */
 public class OASModifier {
     List<Diagnostic> diagnostics = new ArrayList<>();
+    private static final PrintStream outStream = System.out;
```

Comment:
We usually use `System.err`  to print all messages to the console.

Reply:
We have used the `System.out` to print all the diagnostics we get from the client/service generations. This includes warnings and errors.

https://github.com/ballerina-platform/openapi-tools/blob/a9ce3cf2318ba258e2a4b92949341d995e6744c0/openapi-cli/src/main/java/io/ballerina/openapi/cmd/BallerinaCodeGenerator.java#L209

There are instances where we use `System.err` to print the errors. @lnash94 is there any reason behind this decision? Otherwise we can stick to print all the **error** diagnostics using `System.err` and the **warnings** using `System.out`. WDYT?

Reply:
But is it OK to provide the warning messages in a error stream?

Reply:
As per [GNU C Library](https://www.gnu.org/software/libc/manual/html_node/Standard-Streams.html), the error stream is used for error messages and diagnostics issued by the program. So it is okay

Reply:
Ack, will change it to `System.err` 👍🏼

Reply:
Fixed with [21628fe](https://github.com/ballerina-platform/openapi-tools/pull/1766/commits/21628fe6c77a0f54d70984c8e48d6718612b1319)